ch_tools/chadmin/cli/table_group.py

```diff
@@ -220,14 +230,14 @@ def columns_command(ctx, database_name, table_name):
         is_flag=True,
         help="Filter in tables in read-only state only.",
     ),
-    constraint=RequireAtLeast(1),
+    constraint=If(IsSet("detached"), then=accept_none, else_=RequireAtLeast(1)),
 )
 @option(
     "--cluster",
     "--on-cluster",
     "on_cluster",
     is_flag=True,
-    help="Delete tables on all hosts of the cluster.",
+    help="Delete tables on all hosts of the cluster. Doesn't work with --detached flag.",
```

Comment:
> "Doesn't work with --detached flag." 

Is it required? Don't automatic help messages cover option constraints and dependencies?

Reply:
You right. I will remove it.